### PR TITLE
fix bit_rate (MBR, GBR) en/decoding

### LINKED ISSUE #150: [Help] MBR values
I was trying to interconnect the upg-vpp to free5gc which worked fine after some adaptations (and your support). However I wanted to manage the create QER request within the upg but I have an inconsistency with the MBR. When I display it on the CLI (show upf session), it is different from the MBR on the PDU session request. I would like to know how this IE is managed in the upg (encoding etc...). Thank you in advance for any clarification you can give me.

PDU session trace and show upf session command output are attached
[Trace.zip](https://github.com/travelping/upg-vpp/files/6709304/Trace.zip)

### PR DESCRIPTION
**Pull request by RoadRunnr**
The values are actually 40bit (5byte) wide.

resolves #150

### CODE REVIEW

File: upf/pfcp.c [hunk 1]
```diff
@@ -1095,11 +1110,11 @@ decode_bit_rate (u8 * data, u16 length, void *p)
 {
   pfcp_bit_rate_t *v = p;
 
-  if (length < 8)
+  if (length < 10)
     return PFCP_CAUSE_INVALID_LENGTH;
 
-  v->ul = get_u32 (data);
-  v->dl = get_u32 (data);
+  v->ul = get_u40 (data);
```
Comment (ivan4th) [inline upf/pfcp.c:1116]: This is causing build error
```
src/vpp/src/plugins/upf/pfcp.c:1116:11: error: shift count >= width of type [-Werror,-Wshift-count-overflow]
```
apparently due to `data` being `u8 *` upon `(V)[0] << 32` etc

Comment (ivan4th) [reply upf/pfcp.c:1116]: Ref: https://github.com/travelping/upg-vpp/runs/2944778616?check_suite_focus=true#step:3:3127

Comment (RoadRunnr) [reply upf/pfcp.c:1122]: fixed, but I really wonder why this does not happen for u24 getter.

**Review by ivan4th — APPROVED** [review]
LGTM

**Pull request merged**